apps/address-service/domains/common/utils/services/search/searchServiceUtils.js

```diff
@@ -1,43 +1,42 @@
-const { Address, AddressSource } = require('@address-service/domains/address/utils/serverSchema')
-
 /**
  * @param context Keystone context
+ * @param addressGql
+ * @param addressSourceGql
  * @param {{ address: string, key: string, meta: NormalizedBuilding }} addressData
  * @param {string} addressSource
  * @param {{ dv: number, sender: { dv: number, fingerprint: string } }} dvSender
  */
-async function createOrUpdateAddressWithSource (context, addressData, addressSource, dvSender) {
-    const { key, ...addressDataWithoutKey } = addressData
+async function createOrUpdateAddressWithSource (context, addressGql, addressSourceGql, addressData, addressSource, dvSender) {
```

Comment:
GQL objects are strings or results of `gql` functions in `gql.js` modules.
Maybe to rename it to `…ClientUtils` or `…serverUtils` or `…Api`?

Reply:
Done